### ISSUE TITLE
Mobile menu not working

### ISSUE THREAD

**Issue by nickbristow** [18F/pulse#436]
The mobile menu isn't working, also we need to update how its hidden or add a -1 tabindex so the links aren't selected when navigating with a keyboard (when they aren't on screen) 

Note: they should be keyboard accessible when on screen

cc @xtine

**Comment by xtine** [comment]
Mobile menus trigger on staging for me on Chrome device viewer and when I tested with my iPhone 6S.

<img width="359" alt="screen shot 2016-05-24 at 11 41 34 am" src="https://cloud.githubusercontent.com/assets/24054/15515762/900bb402-21a4-11e6-85c3-51700158079d.png">

I'll address the tabindex issue.

**Comment by nickbristow** [comment]
@xtine works for me! 

If you tab through the links after the normal menu, your focus gets lost in the mobile menu. So however you want to stop that is fine.
